src/firefly/js/templates/lightcurve/basic/BasicMissionOptions.js

```diff
@@ -291,7 +334,15 @@ export function basicRawTableRequest(converter, source) {
 }
 
 export function basicOnFieldUpdate(fieldKey, value) {
-    if ([LC.META_TIME_CNAME, LC.META_FLUX_CNAME].includes(fieldKey)) {
+    if ([LC.META_TIME_CNAME].includes(fieldKey)) {
+        const {missionEntries}= getLayouInfo() || {};
+        if (!missionEntries) return;
+
+        if (missionEntries[fieldKey] !== value) {
+            defer(() => makeRawTableRequestByColumnChange(value));
+        }
+        return {[fieldKey]: value};
+    } else if ([LC.META_FLUX_CNAME].includes(fieldKey)) {
```

Comment:
I noticed that this logic is duplicated for all mission type.  Should it be handled at the manager level, then?